store/storetest/session_store.go

```diff
@@ -168,7 +168,7 @@ func testSessionRemoveByUser(t *testing.T, ss store.Store) {
 		}
 	}
 
-	store.Must(ss.Session().PermanentDeleteSessionsByUser(s1.UserId))
+	require.Nil(t, ss.Session().PermanentDeleteSessionsByUser(s1.UserId))
```

Comment:
Just a minor error handling observation, catching the error in a separate line might be more consistent with the rest of the code.

Reply:
@mounicapaladugu Seems reasonable, thanks for the comment!